scripts/cmake/MCWAMP.cmake

```diff
@@ -38,7 +38,8 @@ macro(amp_target name )
   target_include_directories(${name} PRIVATE ${MCWAMP_INC_DIR})
   target_include_directories(${name} SYSTEM INTERFACE $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>)
   add_libcxx_option_if_needed(${name})
-  target_compile_options(${name} PUBLIC -hc -std=c++amp -fPIC)
+  target_compile_options(${name} PUBLIC -hc -fPIC)
+  target_compile_options(${name} PRIVATE -std=c++amp)
```

Comment:
I believe the c++amp is not needed anymore so I'd suggest that we just remove it.

Reply:
Got it, I removed it.